flybywiresim/a32nx
# refactor(ecam): add useSimVars to DOOR/OXY page

Fixes #[issue_no]

## Summary of Changes

DOOR/OXY page modified to utilise new useSimVars function.

## Screenshots (if necessary)



![ecam-door2](https://user-images.githubusercontent.com/5834254/109412557-9c5cfc80-79a0-11eb-95ce-103d7271a513.png)


## References





## Additional context

The changes made should be invisible to the end user.


Discord username (if different from GitHub): tricky_dicky[Z+0]

## Testing instructions

1. Spawn in on stand and open/close the cabin, cargo and catering doors
2. Check in flight that the doors are armed
3. Check that the words CKPT OXY turn amber when the 'Crew Supply' button is off.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
